agent/util-scripts/pbench-register-tool

```diff
@@ -115,17 +115,18 @@ function usage() {
 	printf -- "\tof the file should contain a host name, optionally followed by a label\n"
 	printf -- "\tseparated by a comma (\",\"); empty lines are ignored, and comments are\n"
 	printf -- "\tdenoted by a leading hash, or pound (\"#\"), character.\n"
+	printf -- "\n\t--persistent or --transient can be used to specify tool run type.\n"
 	printf -- "\nAvailable tools:\n"
 	local tool=""
-	python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); [print(f'\t{tool}') for tool in (*meta['transient'].keys(), *meta['persistent'].keys()) ]" ${pbench_bin}/tool-scripts/meta.json
+	python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); print('-Transient:'); [print(f'\t{tool}') for tool in meta['transient'].keys()]; print('-Persistent:'); [print(f'\t{tool}') for tool in meta['persistent'].keys()]" ${pbench_bin}/tool-scripts/meta.json
```

Comment:
I don't want to hold up this PR for this, but, if you find some other reason to make another change....

I really like the headers, but having them preceded by a single hyphen is (as Dave would say) jarring.  You could use the options as headers, in which case they should be preceded by _two_ hyphens (and down-cased).  Or, you could just use spaces.

Also, you can produce the output reasonably straightforwardly with a single `print()` statement:
```
 print('  Transient:', *[f'\t{tool}' for tool in meta['transient'].keys()], '  Persistent:', *[f'\t{tool}' for tool in meta['persistent'].keys()], sep='\n')
```